sphinxcontrib/images.py

```diff
@@ -80,6 +80,15 @@ class ImageDirective(Directive):
     It's backward compatibile and it's adding more cool stuff.
     '''
 
+    align_h_values = ('left', 'center', 'right')
+    align_values = align_h_values
```

Comment:
Do we need two variables here? I see that align_h_values is used only once, in this line.